seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -695,7 +695,12 @@
       "title": "Import a Google Font",
       "description": [
         "Now, let's import and apply a Google font (note that if Google is blocked in your country, you will need to skip this challenge).",
-        "First, you'll need to make a <code>call</code> to Google to grab the <code>Lobster</code> font and load it into your HTML.",
+        "<a href='https://fonts.google.com/' target='_blank'>Google Fonts</a> is a great way to find free fonts to use in a project.",
+        "Once you have selected the preferred font, copy the link tag that is provided and insert it into the <code>head</code> of your page.",
+        "Then, you can make a <code>call</code> to Google to grab the <code>Lobster</code> font and load it into your HTML.",
+        "The font is available to include in your CSS by using the pattern:<br><code>font-family: FAMILY_NAME, GENERIC_NAME;</code>.",
```

Comment:
I recommend changing this to:
>Now you can use the `Lobster` font in your CSS by using `Lobster` as the FAMILY_NAME as in the following example:
`<br><code>font-family: FAMILY_NAME, GENERIC_NAME;</code>`